api/serializers/__init__.py

```diff
@@ -42,6 +42,7 @@ class Meta:
             "last_used_at",
             "num_forwarded",
             "num_blocked",
+            "num_level_one_trackers_blocked",
```

Comment:
Oops, I didn't hit submit on a comment - `num_level_one_trackers_blocked` should be in `read_only_fields`